binaryaudit/cli.py

```diff
@@ -17,9 +17,9 @@
 telemetry_args = arg_parser_telemetry.add_argument_group('telemetry arguments')
 telemetry_args.add_argument('-t', '--enable-telemetry', action='store', required=False,
                             default="n", help="Enable the data storing to telemetry (default: n)")
-telemetry_args.add_argument('-b', '--build-id', action='store', required=False,
+telemetry_args.add_argument('-b', '--build-id', action='store', required=True,
```

Comment:
These are only required if `--enable-telemetry` is passed, please see the `validate_telemetry_args`. The way it is done dynamically as otherwise there might be side effects for other options that might need more flexibility.

Thanks